src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,47 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Manufacturer manufacturer = new Manufacturer("Ford", "USA");
+        manufacturer.setId(1L);
+        Driver jack = new Driver("Jack", "112109");
+        jack.setId(1L);
+        Car car = new Car();
+        car.setModel("Focus");
+        car.setManufacturer(manufacturer);
+        List<Driver> drivers = new ArrayList<>();
+        drivers.add(jack);
+        car.setDriverList(drivers);
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        System.out.println(carService.create(car));
+        System.out.println(carService.get(1L));
```

Comment:
Bohdan says about this at the lecture. Don't use hardcode indexes. Instead of this, create object (write it to DB), and than get by object id